charts/astronomer/templates/houston/api/houston-root-admin-secret.yaml

```diff
@@ -0,0 +1,16 @@
+################################
+## Houston Bootstrap Secrets
+#################################
+kind: Secret
+apiVersion: v1
+metadata:
+  name: astronomer-root-admin-credentials
+  labels:
+    release: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}
+    heritage: {{ .Release.Service }}
+    component: astronomer-root-admin-credentials
+type: Opaque
+data:
+  username: {{ .Values.global.rootAdmin.username | b64enc | quote }}
+  password: {{ randAlphaNum 20 | b64enc | quote }}
```

Comment:
Doesn't `randAlphaNum` change on every deploy, generating a new password every time? Is that the behavior we want?

Reply:
Code on houston side checks if a root admin already exists and if so blocks it.